parser_test.go

```diff
@@ -1812,9 +1812,27 @@ func (s *testParserSuite) TestDDL(c *C) {
 	s.RunTest(c, table)
 }
 
+func (s *testParserSuite) TestHintError(c *C) {
+	parser := New()
+	stmt, warns, err := parser.Parse("select /*+ tidb_unknow(T1,t2) */ c1, c2 from t1, t2 where t1.c1 = t2.c1", "", "")
+	c.Assert(err, IsNil)
+	c.Assert(len(warns), Equals, 1)
```

Comment:
it'e better to further check the warning message

Reply:
added, but it maybe useless, because current parser error message is unreadable and fix that will break many testcase, so I fire another issue to track message content problem at https://github.com/pingcap/parser/issues/86

Reply:
got it.